man/bundle-config.ronn

```diff
@@ -166,6 +166,8 @@ learn more about their operation in [bundle install(1)][bundle-install(1)].
    When set, Gemfiles containing multiple sources will produce errors
    instead of warnings.
    Use `bundle config --delete disable_multisource` to unset.
+* `disable_platform_warnings` (`BUNDLE_DISABLE_PLATFORM_WARNINGS`):
+   Whether Bundler should show platform warnings.
```

Comment:
I would reword this to be a bit more explicit, something like:

```
Disable warnings during bundle install when a dependency is unused on the current platform
```

Reply:
Done :smile: